pkg/mgrconfig/config.go

```diff
@@ -125,6 +125,11 @@ type Config struct {
 	// "android": emulate permissions of an untrusted Android app (supported only on Linux)
 	Sandbox string `json:"sandbox"`
 
+	// This value is passed as an argument to executor and allows to adjust sandbox behavior
+	// via manager config. For example you can switch between system and user accounts based
+	// on this value.
+	SandboxArg int `json:"sandbox_arg"`
```

Comment:
Just curious -- could it happen in the future that we want it to be not just a single number/argument, but a set of sandbox arguments? Woudn't it be easier then to express it all as a string? What will need to be changed in the rest of the implementation and will it remain backward compatible then?

Reply:
As of now we do not care about backward compatibility on Android, because we are the only people customizing executor behavior. So it's not a problem for us to change it. As we learn more about this area we may consider more customization parameters.